modules/glpk/5.0.bcr.3/overlay/BUILD.bazel

```
@@ -0,0 +1,52 @@
+# Copyright 2010-2024 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_workspace_root = package_relative_label("@glpk//:__pkg__").workspace_root
```

```suggestion
_workspace_root = package_relative_label("invalid").workspace_root
```
should work just as well